tests/test_sbt.py

```diff
@@ -471,10 +472,12 @@ def test_sbt_redisstorage():
 
 
 def test_save_zip(tmpdir):
+    # load from zipped SBT, save to zipped SBT, and then search.
     testdata = utils.get_test_data("v5.sbt.zip")
     testsbt = tmpdir.join("v5.sbt.zip")
     newsbt = tmpdir.join("new.sbt.zip")
 
+    # CTB: why do we need to copy, here?
```

Comment:
Avoid concurrent access to file if running `pytest` with `xdist`. And, worst case, also avoid corrupting the original file.